package io.nextweb.promise.exceptions;

public interface ExceptionResult {

	public Object origin();

	public Throwable exception();

}
